<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8C0LO3K" failures="1" tests="2" name="CreateNewsTest" time="35.701" errors="0" timestamp="2024-03-19T20:28:06 IST">
  <testcase classname="com.obsqura.tests.CreateNewsTest" name="createNews" time="10.779">
    <skipped/>
  </testcase> <!-- createNews -->
  <testcase classname="com.obsqura.tests.CreateNewsTest" name="createNews" time="6.246">
    <failure type="java.lang.AssertionError" message="expected [News Created ] but found [News Created Successfully]">
      <![CDATA[java.lang.AssertionError: expected [News Created ] but found [News Created Successfully]
at com.obsqura.tests.CreateNewsTest.createNews(CreateNewsTest.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- createNews -->
</testsuite> <!-- CreateNewsTest -->
